aegisthus-hadoop/src/main/java/com/netflix/Aegisthus.java

```diff
@@ -124,8 +125,20 @@ private void setConfigurationFromCql(Configuration conf) {
             if (pathName.endsWith("-Data.db")) {
                 Path outputPath = new Path(path.getParent(), pathName.replaceAll("[^/]+-Data.db", "*-Data.db"));
                 globs.add(outputPath);
+            } else if (pathName.endsWith("-CompressionInfo.db")) {
+                Path outputPath = new Path(path.getParent(),
+                    pathName.replaceAll("[^/]+-CompressionInfo.db", "*-Data.db"));
+                compressedGlobs.add(outputPath);
             }
         }
+        if (conf.getBoolean(Feature.CONF_ENFORCE_COMPRESSION, false)) {
+            compressedGlobs.retainAll(globs);
+            globs.removeAll(compressedGlobs);
+            if (!globs.isEmpty()) {
+                LOG.warn("Ignoring paths due to lack of compression information: {}", globs);
```

Comment:
do you know what the default aegisthus logger setting is? we'd want to see this pop up in the AM logs, so maybe lower the logging threshold if necessary